Start a test server inside describe (#1068)
cap-js/docs: node.js/cds-test.md

  })
})
```

> > Start a test server with `cds.test(...)` inside a `describe` block of the test, to ensure that tests are running with `jest` and `mocha`.

Reviewer: ```suggestion
> To ensure that your tests run with both `jest` and `mocha`, start a test server with `cds.test(...)` inside a `describe` block of the test.
```
